cost/azure/reserved_instances/utilization/azure_reserved_instance_utilization.pt

```diff
@@ -130,76 +263,198 @@ script "azure_ri_request", type: "javascript" do
-    groupings=_.sortBy(groupings, 'reserved');
+
+    combined_reservation_details = _.reject(reservation_charges_data, function(data){ return data.reservationId == "" })
+    combined_reservation_details = _.reject(combined_reservation_details, function(data){ return Number(data.avgUtilizationPercentage) > param_utilization })
+
+    total_wastage = cur + ' ' + formatNumber((Math.round(total_wastage * 100) / 100), separator)
+
+    all_data = {
+      "reservation_details": combined_reservation_details,
+      "message": "The total estimated monthly wasted spend is " + total_wastage
+    }
   EOS
 end
 
-##########
+###############################################################################
 # Policy
-##########
+###############################################################################
 
-policy "azure_superseded_instances_policy" do
-  validate_each $grouped_utilizations do
-    summary_template "(Enrollment ID: {{parameters.param_enrollment}}): {{ len data }} Underutilized Azure Reserved Instances"
+policy "policy_azure_active_reserved_instances" do
+  validate $ds_reservation_purchase_details do
+    summary_template "(Enrollment ID: {{parameters.param_enrollment_id}}): {{ len data.reservation_details }} active Azure Reserved Instances - {{ parameters.param_lookback_period }}"
+    detail_template <<-EOS
+{{data.message}}
+EOS
     escalate $email
-    check logic_not(lt(to_n(val(item,"util")),$param_utilization))
-    export do
+    check eq(size(val(data, "reservation_details")), 0)
+    export "reservation_details" do
       resource_level true
-      field "id" do
-        label "RI ID"
+      field "reservationId" do
+        label "Reservation ID"
       end
-      field "sku" do
+      field "purchasingSubscriptionGuid" do
+        label "Purchasing Subscription GUID"
+      end
+      field "skuName" do
         label "SKU Name"
       end
-      field "reserved" do
-        label "Reserved Hours"
+      field "region" do
+        label "Region"
       end
-      field "used" do
+      field "term" do
+        label "Term"
+      end
+      field "quantity" do
+        label "Quantity Purchased"
+      end
+      field "eventDate" do
+        label "Purchase Date"
+      end
+      field "eventType" do
+        label "Event Type"
+      end
+      #field "expirationDate" do
+      #  label "Expiration Date"
+      #end
+      field "avgUtilizationPercentage" do
+        label "Avg. Utilizaton %"
+      end
+      field "usedHours" do
         label "Used Hours"
       end
-      field "util" do
-        label "Avg Util Pct"
+      field "reservedHours" do
+        label "Reserved Hours"
+      end
+      field "amount" do
+        label "Reservation Monthly Cost"
+      end
+      field "amountPaidToDate" do
+        label "Reservation Amount Paid to Date"
+      end
+      field "amountOutstandingToDate" do
+        label "Reservation Amount Outstanding"
+      end
+      field "wastageAmount" do
```

Comment:
Just getting to reviewing this, apologies!  Any policy template can generate a spend recommendation now:
https://flexera.atlassian.net/wiki/spaces/PROD/pages/1712000589/Standard+Fields+for+Policy+Incidents

Any thoughts for changing some of these field names to use those standard fields so the incidents show up as a spend recommendation as well @nia-vf1 ?